app/src/main/report/components/ReportsList/ReportsListSorting.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { connect } from 'react-redux';
+import { Dispatch, Action } from "redux";
+
+import {reportsActions} from "../../actions/reportsActions";
+import {ReportsSortingFields} from "../../actionTypes/ReportsActionsTypes";
+
+interface ReportsListSortingProps {
+    sort: (value: ReportsSortingFields) => void;
+}
+
+export const ReportsListSortingComponent: React.StatelessComponent<ReportsListSortingProps> = (props: ReportsListSortingProps) => (
+    <div>
+        <label>Sorting</label>
```

Comment:
Notes for styling this:
1. "Sort by:" rather than "Sorting"
2. wrap `label` and `select` in a `form-group`, wrapped in a `form-inline`
3. give the `select` a class of `form-control`